src/pool/srv_layout.h

```diff
@@ -68,7 +68,7 @@ extern d_iov_t ds_pool_prop_policy;		/* string (tiering policy) */
 extern d_iov_t ds_pool_prop_global_version;	/* uint32_t */
 extern d_iov_t ds_pool_prop_upgrade_status;	/* uint32_t */
 extern d_iov_t ds_pool_prop_upgrade_global_version;/* uint32_t */
-extern d_iov_t ds_pool_prop_scrub_sched;	/* uint64_t */
+extern d_iov_t ds_pool_prop_scrub_mode;	/* uint64_t */
```

Comment:
just clarifying / no change requested - if a pool was previously created with an earlier version of master branch and its rdb has the old key string "scrub_sched", then (after this PR is landed) that pool will probably fail to start (seeing -DER_NONEXIST error) since the new key "scrub_mode" will not be found in the pool_prop_read() execution? I think it's probably OK since the impact is within the same development cycle (master branch toward release 2.4). And that kind of limited impact has occurred other prior commits IIRC.